app/DoctrineMigrations/Version20160822211231.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20160822211231 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE Topic ADD lectureDuration INT DEFAULT NULL, CHANGE lectureFrom lectureDate DATE DEFAULT NULL, DROP lectureUntil');
```

Comment:
Is this right: lectureFrom/Date getting converted to Date? Or did migrations make some weird error? I cannot see any type changes in the entity.